--- a/apps/com.baidu.baidunetdisk/entries/applications/com.baidu.baidunetdisk.desktop	2020-06-24 12:48:38.000000000 +0800
+++ b/apps/com.baidu.baidunetdisk/entries/applications/com.baidu.baidunetdisk.desktop	2020-06-24 12:49:30.968473421 +0800
@@ -3,8 +3,8 @@
 Comment=Baidu Netdisk
 Comment[zh_CN]=百度网盘
 Comment[zh_TW]=百度网盘
-Exec="/opt/apps/com.baidu.baidunetdisk/files/baidunetdisk/baidunetdisk" %U
-Icon=/data/opt/apps/com.baidu.baidunetdisk/entries/icons/hicolor/scalable/apps/baidunetdisk.svg
+Exec=/usr/bin/baidunetdisk %U
+Icon=/usr/share/icons/hicolor/scalable/apps/baidunetdisk.svg
 MimeType=x-scheme-handler/baiduyunguanjia;
 Name=Baidu Netdisk
 Name[zh_CN]=百度网盘
